test/no-empty-file.mjs

```diff
@@ -0,0 +1,37 @@
+import outdent from 'outdent';
+import {getTester} from './utils/test.mjs';
+
+const {test} = getTester(import.meta);
+const space = ' ';
+const tab = '	';
+
+test.snapshot({
+	valid: [
+		'const x = 0;',
+		'\'use strict\'; const x = 0;',
+		';; const x = 0;',
+		'{{{;;\'use strict\'; const x = 0;}}}',
+	],
+	invalid: [
+		'',
+		space,
+		tab,
+		'\n',
+		'\r',
+		'\r\n',
+		outdent`
+
+
+		`,
+		'// comment',
+		'/* comment */',
+		'\'use strict\';',
+		';',
+		';;',
+		';;\'use strict\';',
+		'{}',
```

Comment:
Let's also test `({})`, it should be valid, because it's an object.

Reply:
Done!

Reply:
> it should be valid.

Reply:
My apologies. Should be addressed now.